Submodels/MTSET_call.py

```
@@ -1,63 +1,110 @@
         The defined subprocess has the inputs, outputs, and error file 
         handles sent to PIPE for direct interaction within the Python code.  
         """
 
+        # Get the directory where the current Python file is located
+        current_file_directory = os.path.dirname(os.path.abspath(__file__))
+
+        # Change the working directory to the directory of the current Python file
+        os.chdir(current_file_directory)
+
+        # Generate subprocess
         self.process = subprocess.Popen([self.fpath, self.analysis_name], 
```

Ruff (0.8.2)

`subprocess` call with `shell=True` identified, security issue

(S602)